lib/core/bin/verify-generated-files

```diff
@@ -2,11 +2,12 @@
 set -euo pipefail
 source "$(dirname "${BASH_SOURCE[0]}")/../include/common.bash"
 
-if [[ $# = 0 ]]; then
+files="$(make list-generated --no-print-directory)"
```

Comment:
This doesn't make sense - make targets don't accept arguments.

Reply:
This _does_ work, but `man` does also indicate that options appear before targets.

```suggestion
files="$(make --no-print-directory list-generated)"
```